css/screen/large-screen.css

```diff
@@ -0,0 +1,148 @@
+
+.top-header {
+  background-color: #f72210;
+  height: 90px;
+}
+
+.bottom-header {
+  padding-right: 160px;
+  background-color: #fff;
+  border-bottom-color: #f4f5f6;
+  padding-top: 15px;
+  padding-bottom: 15px;
+  border-bottom-width: 2px;
+  border-bottom-style: solid;
+  z-index: 1;
+}
+
+.bottom-header::before {
+  position: absolute;
+  top: 0;
+  right: 0;
+  content: '';
+  display: block;
+  width: 66px;
+  background-color: #f72210;
+  border-left: 12px solid white;
+  border-top: 13px solid #f72210;
+}
+
+.ul-section {
+  font-family: 'Roboto Condensed', sans-serif;
+  font-size: 19px;
+  width: 100%;
+}
+
+.first {
+  border-left: none;
+}
+
+.leaf {
+  border-left-color: #d9d9d9;
+  flex-grow: 1;
+  text-align: center;
```

Comment:
I think there's a bootstrap class for text align.

Reply:
Sure, i will make the changes immediately